src/components/ShareToken/ShareToken.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+
+import { ReactComponent as ShareIcon } from '../../images/icon-share.svg';
+import { ReactComponent as CopyIcon } from '../../images/icon-copy.svg';
+import './ShareToken.css';
+
+const ShareToken = ({ token }) => {
+  const [showMobileShare, setShowMobileShare] = useState(false);
+
+  function copyToken() {
+    navigator.clipboard.writeText(token).catch((err) => {
+      document.execCommand(token); // possible fallback for older browsers
+      console.log(err);
+    });
+  }
+
+  return (
+    <>
+      <button
+        type="button"
+        onClick={() => setShowMobileShare(!showMobileShare)}
```

Comment:
I'm really liking the ability for it to toggle the hiding and showing on every click!

Reply:
Just wait 'til we get fancy and animate everything!